examples/maxwell2_fdem/emi_tutorial/Response function.ipynb

```diff
@@ -0,0 +1,128 @@
+{
```

Comment:
can you remove the notebook from this repository and put it in: https://github.com/ubcgif/em_examples (then we can hook it up to binders!)